pkg/storj/node.go

```diff
@@ -3,13 +3,174 @@
 
 package storj
 
-import "encoding/hex"
+import (
+	"crypto"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+	"math/bits"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/crypto/sha3"
+	"storj.io/storj/pkg/utils"
+)
+
+// IdentityLength is the number of bytes required to represent a node id
+const IdentityLength = 32
```

Comment:
Replace it with:
```
type NodeID [32]byte

const IdentityLength = len(NodeID{})
```

It keeps the NodeID definition clearer. However, we probably should eliminated IdentityLength altogether, it makes more sense to use `len(nodeID)`.